src/components/HtmlView.js

```diff
@@ -2,23 +2,23 @@ import PropTypes from 'prop-types';
 import React, { memo } from 'react';
 import HTML from 'react-native-render-html';
 import {
-  IGNORED_TAGS,
   alterNode,
-  makeTableRenderer,
+  cssRulesFromSpecs,
   defaultTableStylesSpecs,
-  cssRulesFromSpecs
-} from 'react-native-render-html-table-bridge';
+  IGNORED_TAGS,
+  makeTableRenderer
+} from '@native-html/table-plugin';
 import { WebView } from 'react-native-webview';
 
 import { colors, consts, normalize, styles } from '../config';
 import { imageWidth, openLink } from '../helpers';
 
-const tableCssRules =
+const cssRules =
   cssRulesFromSpecs({
     ...defaultTableStylesSpecs,
     linkColor: colors.primary,
-    // TODO: font family was not working at the moment with following implementation
-    //       https://github.com/jsamr/react-native-render-html-table-bridge/blob/master/readme.md#how-to-load-custom-fonts
+    // TODO: font family documentation
+    //      https://github.com/native-html/plugins/tree/rnrh/4.x#how-to-load-custom-fonts
```

Comment:
i was not able to make it work and asked for more information here https://github.com/native-html/plugins/issues/13#issuecomment-885781518

Reply:
created #293 for it